### PR TITLE
Create and populate the errors directory

### PR DESCRIPTION
**Pull request by Aversiste**
This is taken straight from
https://github.com/devops-coop/ansible-haproxy/pull/104.

Credits to Reza Jelveh AKA @fishman.

### CODE REVIEW

**Comment by Aversiste** [comment]
Would close issue #15.

File: tasks/configure.yml [hunk 1]
```diff
@@ -138,6 +138,48 @@
...
   when: haproxy_userlists is defined
 
+## CREATE ERROR PAGES
+
+- name: Add error directory
+  file:
+    path: "{{ haproxy_config_dir }}/errors"
+    state: directory
+
+- name: '400'
```
Comment (olo-dw) [inline tasks/configure.yml]: The name is too short to me. My advice is to prefix with the tasksbook name `configure`, and to start with _Ensure ..._.

My suggestion is : `- name: Configure - Ensure http 400 error file is present.`

File: tasks/configure.yml [hunk 2]
```diff
@@ -138,6 +138,48 @@
...
+  copy:
+    src: 400.http
```
Comment (olo-dw) [inline tasks/configure.yml]: I am not quite sure that yamllint let it go. I think you should provide quotes.

File: tasks/configure.yml (hunk 1 above)
Comment (Aversiste) [reply tasks/configure.yml]: The taskbook as no prefix right now so I am leaving this part for another PR.

File: tasks/configure.yml (hunk 2 above)
Comment (Aversiste) [reply tasks/configure.yml]: yamllint doesn't complain with the default configuration.

File: tasks/configure.yml (hunk 1 above)
Comment (Aversiste) [reply tasks/configure.yml]: I renamed the tasks.

**Pull request merged**